code/modules/antagonists/wizard/equipment/spellbook_entries/mobility.dm

```diff
@@ -30,6 +30,13 @@
 	spell_type = /datum/action/cooldown/spell/jaunt/ethereal_jaunt
 	category = "Mobility"
 
+/datum/spellbook_entry/swap
+	name = "Swap"
+	desc = "Switch places with any living target."
```

Comment:
You should probably mention that they have to be in sight, first thought I had on reading this was 'anyone on the z level'